specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2462,14 +2512,22 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+			      "UserAssigned"
           ],
           "x-ms-enum": {
             "name": "IdentityType",
             "modelAsString": false
```

Comment:
Suggest to use "modelAsString": true, or it will be breaking change when add new value to the enum, and even will break before API version.
Anyway, Need to check with OneSDK team if this is allowed now, or need a new API version, since it might be a breaking change.

Reply:
Change implemented.